browser/ui/views/infobars/request_otr_infobar.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/infobars/request_otr_infobar.h"
+
+#include <utility>
+
+#include "build/build_config.h"
+#include "ui/views/view.h"
+
+RequestOTRInfoBar::RequestOTRInfoBar(
```

Comment:
Unless I'm missing something, we don't seem to be customizing anything on ConfirmInfoBar, so can we just call `CreateBraveConfirmInfoBar` in `request_otr_infobar_delegate`?

Reply:
Actually looks like you are using that already, so this class isn't needed anymore?

Reply:
removed!